src/vstart.sh

```diff
@@ -713,7 +719,7 @@ EOF
 EOF
 	        fi
 	        prun $SUDO "$CEPH_BIN/ceph-authtool" --create-keyring --gen-key --name="mds.$name" "$key_fn"
-	        ceph_adm -i "$key_fn" auth add "mds.$name" mon 'allow profile mds' osd 'allow *' mds 'allow' mgr 'allow profile mds'
+	        ceph_adm -i "$key_fn" auth add "mds.$name" mon 'allow profile mds' osd 'allow rw tag cephfs metadata=*' mds 'allow' mgr 'allow profile mds'
```

Comment:
Needs access to the data pool as well?

Reply:
Why?

Reply:
PurgeQueue needs to be able to delete inode blocks in the data pool.

Reply:
Oh yeah. Fixed.